### PR TITLE
bug: modifies latency due to flakybot issues

### LINKED ISSUE #775: tests.system.test_gbq.TestReadGBQIntegration: test_timeout_configuration failed
Note: #762 was also for this test, but it was closed more than 10 days ago. So, I didn't mark it flaky.

----

commit: 631532cc3f3728e529da1c555756bfa0e3e20e5b
buildURL: [Build Status](https://source.cloud.google.com/results/invocations/d5bb8c3d-450b-4eeb-bcd4-c19b6644e20a), [Sponge](http://sponge2/d5bb8c3d-450b-4eeb-bcd4-c19b6644e20a)
status: failed

### PR DESCRIPTION
**Pull request by chalmerlowe**
Testing a slightly longer delay for latency due to some flakybot issues.

Fixes #775  🦕

### CODE REVIEW

**Comment by chalmerlowe** [comment]
@Linchin 

This repo fails in the continous testing. I would like to merge this to main (provided all other tests are successful) and let it ruminate for a while to see how things play out.

To that end, I want to:
* Get a release pushed out before we merge this
* Tag the next release candidate as `Do not merge`
* Merge this PR
* Let continuous testing do its thing for a while to see if the same intermittent issue appears in flakybot again
* If we appear to have success remove the `do not merge` tag

**Comment by Linchin** [comment]
SG. I think we need to be careful making a release with this change though, this may break some user code.

**Review by Linchin — APPROVED** [review]

**Pull request merged**
